ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/HelperFunctions.kt

```diff
@@ -14,6 +15,12 @@ import kotlin.coroutines.suspendCoroutine
 @ReadOnlyComposable
 internal fun isInPreviewMode() = LocalInspectionMode.current
 
+@Composable
+@ReadOnlyComposable
+internal fun windowAspectRatio(): Float {
+    return LocalConfiguration.current.screenHeightDp.toFloat() / LocalConfiguration.current.screenWidthDp
```

Comment:
Nice: maybe extract `LocalConfiguration.current` in a `val` here to avoid calling it twice?